### DISCUSSION TITLE
Hacktoberfest 2022

### DISCUSSION THREAD

**Discussion by oliverklee** [MyIntervals/PHP-CSS-Parser#396] [Ideas]
I'd like to propose this project to participate in [Hacktoberfest](https://hacktoberfest.com/), which would allow it to potentially get some contributions and (hopefully) new contributors.

For this, the tag `hacktoberfest` needs to be added to the repository (in the repository settings), and the issue label `hacktoberfest` needs to be created.

Optionally, some easy-for-new-contributors issues can be tagged with the new label, and the maintainer(s) can register at https://hacktoberfest.com/ with their GitHub account as a maintainer.

What do you think, @sabberworm, @westonruter ?

**Comment by sabberworm** [comment]
I’m in favour if it helps with getting high-quality solutions to long-standing issues. If it only attracts people wanting to add features (and, thus, complexity), I don’t know whether it’ll be worth the effort. Do you already have some issues in mind that could be tackled by potential contributors?

**Reply by oliverklee** [reply]
Not yet, but I could have a look at the code and create some easy cleanup tickets for this. Would you like me to do so?

**Reply by sabberworm** [reply]
If you’re willing to set up everything, I can make you a project member. Would that work for you?

**Reply by oliverklee** [reply]
Yes, that would work for me.

**Reply by oliverklee** [reply]
After accepting the invitation (thanks!), I still don't seem to have the necessary permissions for adding the topic. Could you please do that?

1. on the project root page, click on the top-right cogwheel icon
2. add the topic "hacktoberfest"
3. click on "Save changes"
4. Done!

(I was able to create the issue label without problems, though.)

**Comment by oliverklee** [comment]
Closing. We have participated in Hacktoberfest 2022 (without any contributions, though).
